src/components/LHNOptionsList/types.ts

```diff
@@ -0,0 +1,115 @@
+import type {ContentStyle} from '@shopify/flash-list';
+import type {RefObject} from 'react';
+import type {StyleProp, TextStyle, View, ViewStyle} from 'react-native';
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import type {CurrentReportIDContextValue} from '@components/withCurrentReportID';
+import type CONST from '@src/CONST';
+import type {OptionData} from '@src/libs/ReportUtils';
+import type {Locale, PersonalDetailsList, Policy, Report, ReportAction, ReportActions, Transaction} from '@src/types/onyx';
+
+type OptionMode = ValueOf<typeof CONST.OPTION_MODE>;
+
+type LHNOptionsListOnyxProps = {
+    /** The policy which the user has access to and which the report could be tied to */
+    policy: OnyxCollection<Policy>;
+
+    /** All reports shared with the user */
+    reports: OnyxCollection<Report>;
+
+    /** Array of report actions for this report */
+    reportActions: OnyxCollection<ReportActions>;
+
+    /** Indicates which locale the user currently has selected */
+    preferredLocale: OnyxEntry<Locale>;
+
+    /** List of users' personal details */
+    personalDetails: OnyxEntry<PersonalDetailsList>;
+
+    /** The transaction from the parent report action */
+    transactions: OnyxCollection<Transaction>;
+
+    /** List of draft comments */
+    draftComments: OnyxCollection<string>;
+};
+
+type CustomLHNOptionsListProps = {
+    /** Wrapper style for the section list */
+    style?: StyleProp<ViewStyle>;
+
+    /** Extra styles for the section list container */
+    contentContainerStyles?: ContentStyle;
```

Comment:
```suggestion
    contentContainerStyles?: StyleProp<ContentStyle>;
```

Reply:
when using that Flashlist complains that we need to pass `ContentStyle` type

Reply:
You can use `StyleSheet.flatten(contentContainerStyles)` so you will have `ContentStyle` again, and then you can pass to FlashList.